plugins/stk/mallets/mallets.cpp

```diff
@@ -487,7 +499,8 @@ void malletsInstrumentView::changePreset() {
 malletsSynth::malletsSynth(const StkFloat _pitch, const StkFloat _velocity, const StkFloat _control1,
 	const StkFloat _control2, const StkFloat _control4, const StkFloat _control8, const StkFloat _control11,
 	const int _control16, const uint8_t _delay, const sample_rate_t _sample_rate)
-	: m_presetIndex(0) {
+	: m_presetIndex(0)
+{
 	try {
```

Comment:
Again, the opening brace should be on its own line.

Reply:
Fixed, too